### ISSUE TITLE
KITTI Training question

### ISSUE THREAD

**Issue by kjunhwa** [jwchoi384/Gaussian_YOLOv3#68]
Hello, Thank you for providing a nice code in advance.

I read this project's paper named Gaussian YOLOv3: An Accurate and Fast Object Detector Using Localization Uncertainty for Autonomous Driving.

In the KITTI dataset's training, you split the KITTI to train / test in half.

I wonder if the test data was used in the training stage. That is, whether the test data was used as a factor to observe the loss and update the weights.

This question is asked because your mAP is too high for kitti.

**Comment by jwchoi384** [comment]
Hi @kjunhwa 

I didn't use splitted test data in training step. It was only used in testing.
As mentioned in our paper, we split the KITTI training set in half to create training and test data.

If you want to reproduce my results, please refer the link below.
https://github.com/jwchoi384/Gaussian_YOLOv3/issues/45#issuecomment-566437651

**Comment by kjunhwa** [comment]
Thank you for your  fast reply. I understand. Have a good day!
